ElectronClient/app/app.js

```diff
@@ -629,6 +630,7 @@ class Application extends BaseApplication {
 				'',
 				_('Client ID: %s', Setting.value('clientId')),
 				_('Sync Version: %s', Setting.value('syncVersion')),
+				_('Profile Version: %s', BaseModel.db().version()),
```

Comment:
It would be best to use `reg.db()` here because `BaseModel.db()` is a private member of the class, although there's no such concept in JavaScript.

Reply:
Done.